### PR TITLE
edit of tk-auth changes to server docs

### PR DESCRIPTION
**Pull request by gguillotte**
This PR implements the developmental edits by @kayayarai. These edits should be prose-focused, not technical.

### CODE REVIEW

File: documentation/config_file_ca.md [hunk 1]
```diff
@@ -8,33 +8,31 @@ canonical: "/puppetserver/latest/config_file_ca.html"
...
 
 * `authorization-required` determines whether a client certificate is required to access certificate status endpoints. If this parameter is set to `false`, all requests can access this API. If set to `true`, only the clients whose certificate names are included in the `client-whitelist` setting can access the admin API. If this parameter is not specified but the `client-whitelist` parameter is, this parameter's value defaults to `true`.
 * `client-whitelist` contains a list of client certificate names that are whitelisted for access to the certificate status endpoints. Puppet Server denies access to requests at these endpoints that do not present a valid client certificate named in this list.
 
-## Example (Legacy)
+## Example (Deprecated)
 
-If you are not using the new authorization methods, follow this structure to configure  `certificate_status` and `certificate_statuses` endpoint access in `ca.conf`:
+If you are using the deprecated authorization methods, follow this structure to configure  `certificate_status` and `certificate_statuses` endpoint access in `ca.conf`, whitelisting a client named `host`:
```
Comment (camlow325) [inline documentation/config_file_ca.md]: Should either change the reference in the text here to `host1` or in the HOCON example to `host`.

File: documentation/config_file_ca.md [hunk 2]
```diff
@@ -8,33 +8,31 @@ canonical: "/puppetserver/latest/config_file_ca.html"
...
+The `certificate-status` setting in `ca.conf` provides [deprecated][] configuration options for access to the `certificate_status` and `certificate_statuses` HTTP endpoints. These endpoints allow certificates to be signed, revoked, and deleted through HTTP requests, which provides full control over Puppet's ability to securely authorize access. Therefore, you should **always** restrict access to `ca.conf`.
 
-> **Puppet Enterprise Note:** Puppet Enterprise uses these endpoints to provide a console interface for certificate signing. For more information, see the [Certificate Status documentation](/puppet/latest/reference/http_api/http_certificate_status.html).
+> **Puppet Enterprise Note:** Puppet Enterprise uses these endpoints to provide a console interface for certificate signing. For more information, see [Certificate Status ](/puppet/latest/reference/http_api/http_certificate_status.html).
```
Comment (camlow325) [inline documentation/config_file_ca.md]: Cosmetic - extra space before the closing bracket in `[Certificate Status ]`.

**Comment by camlow325** [comment]
:+1: with just a couple of cosmetic notes.  I can merge it after you have a chance to look at those.

**Comment by gguillotte** [comment]
@camlow325 Got 'em.

**Comment by camlow325** [comment]
:+1:

**Pull request merged**